### PR TITLE
Add krishchow as provider-aws maintainer

### LINKED ISSUE #438: Add @krishchow as a provider-aws maintainer
@krishchow has been contributing significant features to `provider-aws` over the past few months. He has also been responsive and reactive to opened issues and big fixes. I believe he has demonstrated all qualities we expect in a [Crossplane maintainer](https://github.com/crossplane/crossplane/blob/master/GOVERNANCE.md#becoming-a-maintainer).

This issue proposes adding @krishchow as a `provider-aws` maintainer. Existing maintainers should indicate votes in the comments.

/cc @negz @muvaf

### PR DESCRIPTION
**Pull request by hasheddan**
### Description of your changes

Adds krishchow as an official provider-aws maintainer.

Signed-off-by: hasheddan <georgedanielmangum@gmail.com>

Fixes #438 

I have:

- [x] Read and followed Crossplane's [contribution process].
- [x] Run `make reviewable test` to ensure this PR is ready for review.

### How has this code been tested

n/a

[contribution process]: https://git.io/fj2m9

### CODE REVIEW

**Comment by hasheddan** [comment]
@jbw976 could you update permissions accordingly after this merges? :)

**Review by jbw976 — APPROVED** [review]
Thanks for all the great contributions so far @krishchow, looking forward to your continued efforts in Crossplane!

**Comment by jbw976** [comment]
@krishchow has been added to this repo with Maintainer permissions 🥇

**Pull request merged**
